### PR TITLE
Fix trip

### PR DESCRIPTION
**Pull request by JackHe313**
fix date display and participants display

### CODE REVIEW

File: src/main/java/com/google/sps/app/servlets/GetTripByID.java [hunk 1]
```diff
@@ -53,7 +53,8 @@ public Trip getTrip(String tripID) {
         Date start = DataStoreHelper.parseDataDate(entity.getString("startDate"));
         Date end = DataStoreHelper.parseDataDate(entity.getString("endDate"));
         List<String> participants = DataStoreHelper.convertToStringList(entity.getList("participants"));
```
Comment (sbahaud) [inline src/main/java/com/google/sps/app/servlets/GetTripByID.java:55]: Is this id list and the names the list of names?

File: src/main/java/com/google/sps/app/servlets/NewTripServlet.java [hunk 2]
```diff
@@ -120,17 +121,17 @@ public Trip getTrip(HttpServletRequest request){
         Date end = DataStoreHelper.parseInputDate(request.getParameter(END_DATE_PARAM));
         String participantInput = StringEscapeUtils.escapeHtml4(request.getParameter(PARTICIPANTS_PARAM));
         List<String> participantIds = new ArrayList<String>();
-        List<String> checkerduplicate = new ArrayList<String>();
+        List<String> names = new ArrayList<String>();
         List<String> participants = DataStoreHelper.splitUserList(participantInput);
```
Comment (sbahaud) [inline src/main/java/com/google/sps/app/servlets/NewTripServlet.java]: Can rename participants to participantNames to be clearer.

File: src/main/java/com/google/sps/app/servlets/NewTripServlet.java [hunk 3]
```diff
@@ -120,17 +121,17 @@ public Trip getTrip(HttpServletRequest request){
...
-        checkerduplicate.add(request.getParameter(USERNAME_PARAM));
+        names.add(request.getParameter(USERNAME_PARAM));
         participantIds.add(request.getParameter(USERID_PARAM));
         for (String name: participants){
```
Comment (sbahaud) [inline src/main/java/com/google/sps/app/servlets/NewTripServlet.java]: Can you fix the indentation for this for loop. seems like there is a few extra tabs

File: src/main/java/com/google/sps/app/servlets/NewTripServlet.java [hunk 4]
```diff
@@ -120,17 +121,17 @@ public Trip getTrip(HttpServletRequest request){
         Date end = DataStoreHelper.parseInputDate(request.getParameter(END_DATE_PARAM));
         String participantInput = StringEscapeUtils.escapeHtml4(request.getParameter(PARTICIPANTS_PARAM));
         List<String> participantIds = new ArrayList<String>();
-        List<String> checkerduplicate = new ArrayList<String>();
+        List<String> names = new ArrayList<String>();
```
Comment (sbahaud) [inline src/main/java/com/google/sps/app/servlets/NewTripServlet.java]: Can you name these existingParticipants and newParticipants. It is slightly confusing currently.

**Review by sbahaud — APPROVED** [review]
Approving with some comments

File: src/main/java/com/google/sps/app/servlets/GetTripByID.java (hunk 1 above)
Comment (JackHe313) [reply src/main/java/com/google/sps/app/servlets/GetTripByID.java:55]: yes

**Pull request merged**